src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/specs/UploadSide.spec.js

```diff
@@ -117,29 +147,14 @@ describe('UploadVideo > UploadSide', () => {
       expect(wrapper.emitted().confirm).toBeUndefined();
     });
 
-    it('should show error when upload', async () => {
```

Comment:
and how we process errors now, if we have they?

Reply:
done

did extend state label

Reply:
green error, are you sure?
![image](https://user-images.githubusercontent.com/756731/89790232-4f135d80-db2a-11ea-835d-276614fc33af.png)

Reply:
really, text so small that color wasn't noticed lol

Reply:
done